build/travis/job_macos_lupdate/script.sh

```diff
@@ -9,12 +9,15 @@ fi
 
 # Translation routines
 # update translation on transifex
+# remove obsolete strings
+OBSOLETE=-no-obsolete # '-noobsolete' in older QT versions
```

Comment:
We use 5.9.8 for macOS builds.

Reply:
Yes, that 'older' referres to 'much older, Qt 4.8 or some such